requirements.txt

```diff
@@ -1,18 +1,17 @@
 # Requirements for production / heroku
 
 # Django
-# 1.8 is waiting on django-simple-history
-Django==1.7.8  # rq.filter: >=1.7, <1.8
+Django==1.7.10  # rq.filter: >=1.7, <1.8
 
 # Better templates
 MarkupSafe==0.23
-Jinja2==2.7.3
+Jinja2==2.8
```

Comment:
Are you sure that works in practice? We had a few issue with enforced encoding in kuma.